lighthouse-cli/test/smokehouse/perf/expectations.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * Expected Lighthouse audit values for byte efficiency tests
+ */
+module.exports = [
+  {
+    initialUrl: 'http://localhost:10200/online-only.html',
+    url: 'http://localhost:10200/online-only.html',
+    audits: {
+      'speed-index-metric': {
+        score: 100,
+        extendedInfo: {
+          value: {
+            timings: {
+            }
+          }
+        }
+      },
+      'time-to-interactive': {
+        score: 100,
+        extendedInfo: {
+          value: {
+            foundLatencies: {
```

Comment:
hang on I moved this to `latencies.timeToInteractive` in my PR and I just pushed it but you already merged 😬 

do people rely on this?